kpi/models/import_export_task.py

```diff
@@ -357,21 +366,23 @@ def _parse_b64_upload(self, base64_encoded_upload, messages, **kwargs):
             else:
                 asset = destination
                 asset.content = survey_dict
-                server = settings.KOBOCAT_URL
+
+                # what if owner is an organization
```

Comment:
I don't believe Kobo has the concept of organisations.
Or what was this in regards to?

Reply:
If at some point we wanted an organization to own assets (which I for-see a use case in the future ), i'd like to think we'd have to consider updating this bit of the code to meet those needs. Before then, we can consider this just a heads-up comment :-)